### DISCUSSION TITLE
Erro 404

### DISCUSSION THREAD

**Discussion by karen-regina** [community/community#138528] [Other Feature Feedback, Questions, & Ideas]
### Body

Como resolve esse erro??
![Captura de tela_12-9-2024_201524_karen-370 github io](https://github.com/user-attachments/assets/c69286e2-aa48-4ac5-a383-a9ebb0b97b63)

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by phuyalgaurav** [comment]
You need to have a index.html file. Rename the file that you want to see as the website to `index.html`.

**Comment by ebndev** [comment]
Hi @Karen-370, thanks for posting in GitHub Discussions! 

I've gone ahead and moved this post to our `Pages` category, as this topic may be more relevant there and might help someone give you a nudge in the right direction. 😁

In the meantime, please feel encouraged to explore existing discussions in the category and utilize the search in case other users may have run into the same issue!

**Comment by iMohitNayal** [comment]
You may want to go ahead and move this post to our pages category, Pages as this topic may be more relevant there and help guide someone in the right direction.

**Comment by sasan-farhadi** [comment]
Create the index.html file in the main root of your project and test again.
